features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageTypeResolverTest.kt

```diff
@@ -49,7 +50,9 @@ internal class ImageTypeResolverTest {
         @IntForgery(min = 0, max = IMAGE_DIMEN_CONSIDERED_PII_IN_DP) fakeHeight: Int
     ) {
         // When
-        val result = testedTypeResolver.isDrawablePII(mockBitmapDrawable, fakeWidth, fakeHeight)
+        whenever(mockBitmapDrawable.intrinsicWidth).thenReturn(fakeWidth)
+        whenever(mockBitmapDrawable.intrinsicHeight).thenReturn(fakeHeight)
+        val result = testedTypeResolver.isDrawablePII(mockBitmapDrawable, density = 1f)
```

Comment:
minor: here and below it is better to put test setup under `Given` block 

```suggestion
        // Given
        whenever(mockBitmapDrawable.intrinsicWidth).thenReturn(fakeWidth)
        whenever(mockBitmapDrawable.intrinsicHeight).thenReturn(fakeHeight)
        
        // When
        val result = testedTypeResolver.isDrawablePII(mockBitmapDrawable, density = 1f)
```